remote/remote.go

```diff
@@ -207,8 +241,8 @@ func prepareNewWindowsImage(ri *Image) error {
 	return nil
 }
 
-func newV1Image(keychain authn.Keychain, repoName string, platform imgutil.Platform) (v1.Image, error) {
-	ref, auth, err := referenceForRepoName(keychain, repoName)
+func newV1Image(keychain authn.Keychain, repoName string, platform imgutil.Platform, reg *registry) (v1.Image, error) {
```

Comment:
```suggestion
func newV1Image(keychain authn.Keychain, repoName string, platform imgutil.Platform, reg registry) (v1.Image, error) {
```
Does it need to be a pointer? Then you could avoid the nil check.